### PR TITLE
Add Storefront Purchase and Purchase Preview

### LINKED ISSUE #127: Purchase and PreviewOrder Storefront API definitions
- `PostPurchase`
- `PostPreviewOrder`

### PR DESCRIPTION
**Pull request by justnero**
This would allow customers to complete the checkout form

- [StorefrontPostPurchase](https://preview.redoc.ly/rebilly-storefront/storefront-purchase/tag/Purchase#operation/StorefrontPostPurchase)
- [StorefrontPostPreviewPurchase](https://preview.redoc.ly/rebilly-storefront/storefront-purchase/tag/Purchase#operation/StorefrontPostPreviewPurchase)

Closes #127

### CODE REVIEW

**Review by ashkarpetin — DISMISSED** [review]

**Comment by mattsanf** [comment]
https://preview.redoc.ly/rebilly-storefront/storefront-purchase/tag/Purchase#operation/StorefrontPostPurchase

Seems incomplete. 

How do we know what products & plans are being purchased?
How do we know the billing & shipping address of the purchase?

**Comment by mattsanf** [comment]
Also the preview response is missing what currency the amounts are in

**Comment by justnero** [comment]
Thank you @mattsanf, fixed

**Comment by mattsanf** [comment]
Looking at the preview billing addresses. Can phoneNumber and emails be empty arrays? everything else is nullable (I'm assumin gyes)

**Comment by justnero** [comment]
Yes, they can be empty

**Review by mattsanf — DISMISSED** [review]

**Review by mattsanf — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/combined.yaml [hunk 1]
```diff
@@ -234,6 +234,7 @@ tags:
   - name: Webhook Credentials
   - name: Checkout Forms
   - name: Checkout Pages
+  - name: Checkout Forms
```
Comment (ashkarpetin) [inline openapi/combined.yaml:237]:
```suggestion
  - name: Checkout Forms
```
duplicate?

**Review by ashkarpetin — APPROVED** [review]

**Review by mattsanf — APPROVED** [review]

**Pull request merged**